### ISSUE TITLE
Should it support pydoc

### ISSUE THREAD

**Issue by hamishwillee** [ArduPilot/pymavlink#204]
THere is an old set of docs on mavlink.org that states that docs can be generated for mavgen output using pydoc. I tried this and got an error: 
```
ubuntu@ubuntu:~/github/mavlink$ pydoc -w POUT.py
problem in POUT - <type 'exceptions.ValueError'>: Attempted relative import in non-package
```

**Comment by lightmare** [comment]
This appears to have been inadvertently fixed by: #441

**Comment by peterbarker** [comment]
Yay happy accidents.  Thanks @lightmare
